libs/coin-modules/coin-stellar/src/types/bridge.ts

```diff
@@ -32,7 +32,13 @@ export enum NetworkCongestionLevel {
   HIGH = "HIGH",
 }
 
-export const StellarMemoType = ["NO_MEMO", "MEMO_TEXT", "MEMO_ID", "MEMO_HASH", "MEMO_RETURN"];
+export const StellarMemoType = [
+  "NO_MEMO",
+  "MEMO_TEXT",
+  "MEMO_ID",
+  "MEMO_HASH",
+  "MEMO_RETURN",
+] as const;
```

Comment:
Maybe each elem could be part of an enum to be more accurate ? and more easy ti use in other component ? like in 

`const type = memoType === "NO_MEMO" && value ? "MEMO_TEXT" : memoType;` ?

Reply:
Yeah initially I created a type for this but it required more changes than I was confortable making to the Stellar code.
E.g in some places `memoType` is assigned a `value.toString()`.
![image](https://github.com/user-attachments/assets/dc58d56f-68cc-41f6-99ad-9726126583be)

I'm not sure whether there's a good reason for this but rather than risking breaking things I decided not to touch the type of the Stellar transaction and just base my changes on this `StellarMemoType` list.